docs/_includes/common/radio.html

```diff
@@ -129,7 +165,7 @@ <h3 id="radio-group">Grouped Radio</h3>
     <legend>Choose an Option</legend>
     <span class="field">
         <span class="field__control radio">
-            <input class="radio__control" id="group-radio-1" type="radio" value="1" name="radio-group" />
+            <input class="radio__control" id="group-radio-1" type="radio" value="1" />
```

Comment:
Why was the radio group removed?

Reply:
Oh, it's just the `name` attribute example code. I try to remove anything that might be superflous. But now I think about it, I should probably leave it in. It's kind of important! :-)